### ISSUE TITLE
update all the requirements

### ISSUE THREAD

**Issue by willkg** [mozilla/antenna#79]
Update all the requirements that are stale.

I know at least gunicorn has had a release since we codified the requirements list. There are probably others.

**Comment by willkg** [comment]
Out of date requirements:

```
gevent (1.1.1) is 107 days out of date. Latest is 1.1.2
greenlet (0.4.9) is 286 days out of date. Latest is 0.4.10
gunicorn (19.5.0) is 11 days out of date. Latest is 19.6.0
python-mimeparse (1.5.2) is 173 days out of date. Latest is 1.6.0
boto3 (1.4.0) is 64 days out of date. Latest is 1.4.1
botocore (1.4.59) is 13 days out of date. Latest is 1.4.64
s3transfer (0.1.6) is 7 days out of date. Latest is 0.1.8
alabaster (0.7.8) is 80 days out of date. Latest is 0.7.9
argh (0.26.1) is 559 days out of date. Latest is 0.26.2
coverage (4.0.3) is 245 days out of date. Latest is 4.2
flake8 (2.5.4) is 178 days out of date. Latest is 3.0.4
mccabe (0.4.0) is 186 days out of date. Latest is 0.5.2
pbr (1.9.1) is 39 days out of date. Latest is 1.10.0
pluggy (0.3.1) is 374 days out of date. Latest is 0.4.0
pyflakes (1.0.0) is 347 days out of date. Latest is 1.3.0
pytest (2.9.1) is 195 days out of date. Latest is 3.0.3
pytz (2016.4) is 163 days out of date. Latest is 2016.7
PyYAML (3.11) is 885 days out of date. Latest is 3.12
Sphinx (1.4.1) is 172 days out of date. Latest is 1.4.8
virtualenv (15.0.1) is 140 days out of date. Latest is 15.0.3
```

Going through them all and updating them now.

**Comment by willkg** [comment]
Fixed in PR #100.
